### ISSUE TITLE
Add phone sensor compass ?

### ISSUE THREAD

**Issue by GuillaumeJeu** [mpetroff/pannellum#1076]
Hello I would like to know what code should you add to have the compass on the phone which allows you to move with the phone sensors, I saw it on the official site it works but on my own site I don't have it ?

Thank You
![Screenshot_20220211-212951](https://user-images.githubusercontent.com/77529874/153666005-8486d07c-267b-4331-b858-94d30dafdc92.png)
![Screenshot_20220211-213018](https://user-images.githubusercontent.com/77529874/153666010-e7cbc3ae-aa77-4cdf-8173-afccb0e76bc6.png)

**Comment by mpetroff** [comment]
Are you using HTTPS? Browsers disable access to device orientation data on unencrypted HTTP connections for security reasons.

**Comment by GuillaumeJeu** [comment]
I've been trying to establish https for quite some time but I can't. Is there a way around this? (I'm on apache)

**Comment by mpetroff** [comment]
No, it's a browser security restriction. There's no way around it.
